sdk/search/azure-search/src/test/java/com/azure/search/SearchServiceTestBase.java

```diff
@@ -656,4 +660,22 @@ ServiceStatistics getExpectedServiceStatistics() {
             .setCounters(serviceCounters)
             .setLimits(serviceLimits);
     }
+
+    static boolean liveMode() {
+        return setupTestMode() == TestMode.LIVE;
+    }
```

Comment:
Should this be replaced with the `isLiveMode` method in `InterceptorManager`?

Reply:
Yes. Try to replace with the one.

Reply:
My concern is next release will depend on azure-core-test. So I put a helper method currently in search. Will move to interceptorManager.isLiveMode once azure-core-test is released.